1-mandatory/3-reading-list.js

```diff
@@ -48,19 +81,43 @@ As an example for this exercise, you might do the following steps
 - All tests should turn green!!
 **/
 
+ 
+  function logBooks() {
+  return books.forEach((book) => {
```

Comment:
This code works fine but just a minor comment to keep it cleaner. In the test the invocation of the function is:
`logBooks(); `
So it's just invoking the function and not expecting any return value.